mm/ktsan/thr.c

```diff
@@ -38,9 +39,13 @@ kt_thr_t *kt_thr_create(kt_thr_t *thr, int kid)
 		pr_err("KTSAN: maximum number of threads is reached\n");
 		BUG();
 	}
+	/* Kernel does not assign PIDs to some threads, give them fake
+	 * negative PIDs so that they are distinguishable in reports. */
+	if (pid == 0)
+		pid = --pool->new_pid;
```

Comment:
Oh, I meant the one that's spawned before init.
I'm not sure what it's called.
Though now I'm thinking that it might not actually do something meaningful, maybe it also becomes an idle task.
OK, I guess it will make sense to try to fix this if we see reports with negative PIDs, but not from idle threads.